Misc/NEWS.d/next/Library/2024-06-26-16-16-43.gh-issue-121011.qW54eh.rst

```diff
@@ -0,0 +1,2 @@
+:func:`math.log` now supports arbitrary large integer-like arguments in the
```

Comment:
Rather index-like, isn't?

Reply:
"index-like", "integer-like", and "int-like" occur about the same number of times. There is no clear winner. Mark used "integer-like" in its comment.